include/dartpy/pointers.h

```diff
@@ -9,6 +9,7 @@
 #endif
 
 #include "get_signature.h"
+#include <dart/dynamics/dynamics.hpp>
```

Comment:
I think this might be a duplicate now, since we include `dart::dynamics` on the very next line.